extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ScenarioActivity.scala

```diff
@@ -47,9 +50,9 @@ object ScenarioComment {
       lastModifiedAt: Instant,
   ) extends ScenarioComment
 
-  final case class Deleted(
-      deletedByUserName: UserName,
-      deletedAt: Instant,
+  final case class NotAvailable(
```

Comment:
I'm not sure if it's easy to grasp when is not available comment? :D Especially when it has user and modification :D

Reply:
Changed, ScenarioComment with 2 variants: WithContent, WithoutContent